x-pack/plugins/cases/public/components/callout/translations.ts

```diff
@@ -7,15 +7,6 @@
 
 import { i18n } from '@kbn/i18n';
 
-export const READ_ONLY_FEATURE_TITLE = i18n.translate('xpack.cases.readOnlyFeatureTitle', {
```

Comment:
heads up, we're going to have a few conflicts between this and my Cases Cleanup PR #102103